src/components/CreateWallet/index.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable no-console */
+import React, { Component } from 'react';
+import propTypes from 'prop-types';
+import { observer, inject } from 'mobx-react';
+import { NavLink } from 'react-router-dom';
+import Container from '../Container';
+import Header from '../Header';
+import FormBlock from '../FormBlock';
+import Heading from '../Heading';
+import { Password, BackIcon } from '../Icons';
+import Input from '../Input';
+import { Button, IconButton } from '../Button';
+import Loader from '../Loader';
+
+import styles from '../Login/Login.scss';
+
+@inject('userStore', 'appStore')
+@observer
+class CreateWallet extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+  }
+
```

Comment:
If you dont use state, maube you dont need one?

Reply:
now this component using state